docs/config/README.md

```diff
@@ -168,6 +168,23 @@ module.exports = {
 }
 ```
 
+#### Modifying Loader with GraphQL example
```

Comment:
Maybe this title should be `Adding a new loader`.

Reply:
Yeah :) 
sorry, 
I just wanted to help those who want to use other loaders. 
I did not know the right place to add